lib/commands/find.js

```diff
@@ -43,7 +43,8 @@ export async function _getXPathNodes(strategy, selector, mult, context) {
   this._cachedSourceDirty = true;
   const source = await this.getPageSource();
   const doc = new DOMParser().parseFromString(source, 'text/xml');
-  return xpath.select(selector, doc);
+  const single = false;
+  return xpath.select(selector, doc, single);
```

Comment:
It seems like explicit `single: false` helps the readability
https://github.com/goto100/xpath/compare/v0.0.32...0617ceffaa782f76c39df8478b5ab505edeb4e2f